apps/yggdrasil/test/test_helper.exs

```diff
@@ -1,4 +1,8 @@
 ExUnit.start()
 
-Mix.Task.run "ecto.create", ~w(-r Yggdrasil.Repo --quiet)
-Mix.Task.run "ecto.migrate", ~w(-r Yggdrasil.Repo --quiet)
+repo = ~w(-r Yggdrasil.Repo --quiet)
+
+Mix.Task.run "ecto.drop", repo
+Mix.Task.run "ecto.create", repo
+Mix.Task.run "ecto.migrate", repo
+Mix.Task.run "run", ~w(apps/yggdrasil/priv/repo/seeds.exs)
```

Comment:
Awesome, I figured there had to be a way to get the app's path so do appreciate you digging that up. I have been trying to knock out the other stuff and just had not got around to it. I think that second form is exactly what we want.

I also know the drop is still there, but just haven't had time to get back to it. I'll sort that out at some point.